src/scene/vcFlythrough.cpp

```diff
@@ -254,6 +254,17 @@ void vcFlythrough::HandleSceneEmbeddedUI(vcState *pProgramState)
 
   if (m_state != vcFTS_Exporting)
   {
+    if (ImGui::BeginCombo(vcString::Get("flythroughExportFormat"), GetImageSaveFormatExtension(pProgramState->screenshot.imageSaveFormat)))
```

Comment:
This key doesn't seem to be in the translation file?

Reply:
A lot of the PR's I make won't have things in the translation file. Like Pauls first FlyThrough commit. One of the bugs I gotta fix is to put everything to do with FlyThroughs in the translation file, so I'll do all that at once